.pre-commit-config.yaml

```diff
@@ -22,6 +22,11 @@ repos:
     hooks:
       - id: yamllint
         args: ["--strict"]
+  - repo: https://github.com/pre-commit/mirrors-clang-format
```

Comment:
Some questions about how this will affect the development process.

* does this require installing `clang` separately, or will `pre-commit` set that up for you?
* does this work on Windows?